File: ts/filter-content/filtering_get_items_by_linked_item.ts
// Gets a list of items where the 'my_page' item is used in the 'navigation' element
const itemsLinkedToPage = await deliveryClient.items()
  .containsFilter('elements.navigation', ['my_page'])
  .toPromise();

// Gets items linked to at least Jane, John, or both
const itemsWithSpecificAuthor = await deliveryClient.items()  
  .anyFilter('elements.author', ['jane_doe', 'john_wick'])
  .toPromise();

  // Gets pages linking travel insurance as their subpage
const itemsLinkedToSubpage = await deliveryClient.items()
.containsFilter('elements.subpages', ['travel_insurance'])
.toPromise();

// Gets pages linking at least travel insurance, car insurance, or both as their subpage
const itemsLinkedToSubpages = await deliveryClient.items()
.anyFilter('elements.subpages', ['travel_insurance', 'car_insurance'])
.toPromise();

File: ruby/filter-content/filtering_get_items_by_linked_item.rb
# Gets a list of items where the 'my_page' item is used in the 'navigation' element.
delivery_client.items('elements.navigation'.contains('my_page'))

# Gets items linked to at least Jane, John, or both.
delivery_client.items('elements.author'.any('jane_doe', 'john_wick'))

# Gets pages linking travel insurance as their subpage.
delivery_client.items('elements.subpages'.contains('travel_insurance'))

# Gets pages linking at least travel insurance, car insurance, or both as their subpage.
delivery_client.items('elements.subpages'.any('travel_insurance', 'car_insurance'))